cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
Docker exec a watcher that lists files that change. Docker cp them out when they show up in the list. The tech there should be trivial.